### PR TITLE
Add with_delimiter for list method

### PR DESCRIPTION
**Pull request by giangndm**
This PR added missing with_delimiter option for list method

### CODE REVIEW

**Review by guerinoni — APPROVED** [review]

File: src/actions/list_objects_v2.rs [hunk 1]
```diff
@@ -117,6 +117,19 @@ impl<'a> ListObjectsV2<'a> {
...
     }
 
+    /// A delimiter is a character that you use to group keys.
+    ///
+    /// See https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html#API_ListObjectsV2_RequestSyntax for more infos.
+    /// # Example
+    /// ```
+    /// # let bucket = rusty_s3::Bucket::new(url::Url::parse("http://rusty_s3/").unwrap(), rusty_s3::UrlStyle::Path, "doggo", "doggoland").unwrap();
+    /// let mut list = bucket.list_objects_v2(None);
+    /// list.delimiter("/");
```
Comment (paolobarbolini) [inline src/actions/list_objects_v2.rs:127]:
```suggestion
    /// list.with_delimiter("/");
```

**Review by paolobarbolini — APPROVED** [review]
Thank you for the PR. The CI has caught a typo in the docs, which I've fixed in the attached suggestion.

File: src/actions/list_objects_v2.rs (hunk 1 above)
Comment (giangndm) [reply src/actions/list_objects_v2.rs]: @paolobarbolini I fixed as you suggested.

**Comment by paolobarbolini** [comment]
Thanks for the contribution!

**Pull request merged**